packages/cicero-common/models/contract.cto

```diff
@@ -0,0 +1,51 @@
+namespace org.accordproject.contract
+
+/**
+ * Contract data
+ * -- This is the result of matching a template
+ */
+
+/* A state is a concept with a contract status */
+concept State {
+}
+/* A contract is a concept -- This contains the contract data */
+concept Contract {
+}
+/* A clause is a concept -- This contains the clause data */
+concept Clause {
+}
+
+/**
+ * Contract API
+ * -- Describes input and output of calls to a contract's clause
+ */
+
+/* A request is a transaction */
+transaction Request {}
+/* A response is a transaction */
+transaction Response {}
+/* An Error is a concept */
+concept ErrorResponse {}
+/* An event is a transaction */
+transaction Action {}
```

Comment:
Let's discuss tradeoff between using a composer `transaction` vs an `event` type for this.